src/message/html/HtmlNodeTag.js

```diff
@@ -37,15 +37,18 @@ const specialTags = {
   'td': HtmlTagDiv,
 };
 
+const stylesFromClassNames = (classNames = '', styleObj) =>
+  classNames.split(' ').map(className => styleObj[className]);
```

Comment:
We should ignore classes that don't exist locally instead of throwing an exception